Packs/CortexXDR/ReleaseNotes/5_1_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR Malware - Incident Enrichment
+
+- Added tasks to extract usernames from domain usernames in order to allow enrichment with both integrations that domain usernames and integrations that allow only username enrichment.
```

Comment:
I think you are missing the word "allows" after the word "that" in this part:
"...with both integrations that domain usernames and integrations..."